src/core/public/chrome/ui/header/header.tsx

```diff
@@ -669,7 +663,7 @@ export function Header({
 
   return (
     <>
-      {isBannerEnabled && <div id="pluginGlobalBanner" />}
+      {globalBanner && <div className="globalBanner">{globalBanner.component}</div>}
```

Comment:
@sidwang42 Can we make this  header banner JSX a react component, and import here for better separation and extensibility?